samples/openthread/common/overlay-ble.conf

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Enable Bluetooth LE controller which supports multiprotocol
+CONFIG_BT_LL_SOFTDEVICE_DEFAULT=y
+
+# Enable and configure Bluetooth LE
+CONFIG_BT=y
+CONFIG_BT_PERIPHERAL=y
+CONFIG_BT_DEVICE_NAME="NCS DEVICE"
```

Comment:
This means that if the user apply this Kconfig fragment to 2 thread devices for testing, then both shows up with identical names.

To me that indicates the samples that should support multiprotocol should have their own fragment so they are easier to identify, like `ncs coap client`.
Btw. Why only upper case in `NCS DEVICE` ?

Reply:
> Btw. Why only upper case in NCS DEVICE ?
This is taken from original file why it was like that there I don't know.